cmd/utils/nodecmd/defaultcmd.go

```diff
@@ -120,7 +120,7 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 	}()
 
 	if utils.NetworkTypeFlag.Value == SCNNetworkType && utils.ServiceChainConsensusFlag.Value == "clique" {
-		startServiceChainService(ctx, stack)
+		logger.Crit("using clique consensus type is not allowed anymore!")
```

Comment:
Should you use the `if utils.ServiceChainConsensusFlag.Value == "clique" {` condition if you want to give an error when you try to use "clique"?

Reply:
@nogang The purpose of this change is to remove the usage of `ServiceChain`, clique related codes will be removed later.